spec/cucumber/listener/rerun_formatter_spec.js

```diff
@@ -37,7 +38,7 @@ describe("Cucumber.Listener.RerunFormatter", function () {
       });
 
       it("logs nothing", function () {
-        expect(rerunFormatter.getLogs()).toEqual('path/to/scenario:1');
+        expect(rerunFormatter.getLogs()).toEqual(path.join('path/to/scenario:1'));
```

Comment:
unless we are actually passing multiple paths, I don't think using `path.join` makes sense. 
I would suggest using path.join like this:
```js
path.join('path', 'to', 'scenario:1')
```
Conversely we could just use `path.normalize` here. [docs](https://nodejs.org/api/path.html#path_path_normalize_path)